### PR TITLE
EvianTan - update the readme files

### PR DESCRIPTION
**Pull request by EvianTan**
# Description
Due to the dated Readme file, I plan to update it and also create "good" documentation in the near future

## Related PRS (if any):
None

## Main changes explained:
Update the markdown file and upload images

## How to test:
just see the view from this link - https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/tree/Yiyun-readme_updates

## Screenshots or videos of changes:
image
image
image

### CODE REVIEW

**Review by bienzguoa — COMMENTED** [review]
Hi,
It looks good! Are the contents of Roadmap, License and acknowledgement still in updating?

**Review by yufeiStella — COMMENTED** [review]
The readme file is looking good so far!

**Review by mahamakhan — APPROVED** [review]
The readme looks good.

**Review by KurtisIvey — CHANGES_REQUESTED** [review]
I think in line 85 it may be better to mention that there's a preferred node version. Lots of people run into the gyp issue.

1. Node v12.5(v14.17.6 preferred) or higher installed on your machine.

**Review by KurtisIvey — CHANGES_REQUESTED** [review]
I think in line 85 it may be better to mention that there's a preferred node version. Lots of people run into the gyp issue.

1. Node v12.5(v14.17.6 preferred) or higher installed on your machine.

File: README.md [hunk 1]
```diff
@@ -1,63 +1,165 @@
...
+Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
 
-```
-REACT_APP_APIENDPOINT=<put apiendpoint here> npm start
-```
+If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also simply open an issue with the tag "enhancement".
+Don't forget to give the project a star! Thanks again!
 
-For a windows user, you can run
+1. Join One Community as a member
```
Comment (leonzh2k) [inline README.md:126]: Would be helpful to include a link to the volunteer application here

**Review by leonzh2k — COMMENTED** [review]
Are you intending for this new README to be a place where new volunteers can learn how to get the app running? If so, I think the google doc does a better job currently (mentions what to put in `.env` file, etc.

**Comment by eduhorta** [comment]
Not sure if this is outside of the scope of the PR, but, there was a comment on instructions for running the app regarding Apple M1 incompatibility made by @stevenpaalz that could be useful aswell.

image

**Comment by cvtqx** [comment]
Looks good so far!

**Review by Chuehleo — APPROVED** [review]
The readme file looks pretty good!

**Review by wang9hu — APPROVED** [review]
Looks a lot more professional, and I agree with @leonzh2k, this can be where it starts, and gradually transfer from google doc.

**Comment by rachnapandya** [comment]
the readme file is looking great so far with a good format.

**Review by BaileyM96 — CHANGES_REQUESTED** [review]
This README is looking great so far! I think there could be a change that can help out a new volunteer.

## Issue

1.  I think the Installation needs more to get the new volunteer set up correctly and get their user accounts going.

## Fix

1.Have the volunteer application in the contact section and tell the user to fill out the application so they can have google doc link to get correctly set up

**Comment by one-community** [comment]
> ## Fix
> 1.Have the volunteer application in the contact section and tell the user to fill out the application so they can have google doc link to get correctly set up

What if we just linked to our website application, so they are always linking to the most current information

**Comment by BaileyM96** [comment]
> 

Yeah we can do that so we can keep track who is working on the repos and the google docs set up has really good documentation on getting set up

**Review by HectorAgudelo — APPROVED** [review]
looks good so far.

**Review by eduvarjaor — APPROVED** [review]
The readme file looks awesome!

**Review by yurisokolovicz — APPROVED** [review]
The README.md looks good. It contains everything we need to get in touch and start volunteering.

**Review by Indujaks — APPROVED** [review]
The Readme looks great!

**Review by ThapeloMasasa — APPROVED** [review]
The readme is well written and looks good

**Pull request merged**